numba_dpex/core/runtime/context.py

```diff
@@ -21,7 +21,9 @@ def _check_null_result(func):
         @functools.wraps(func)
         def wrap(self, builder, *args, **kwargs):
             memptr = func(self, builder, *args, **kwargs)
-            msg = "USM allocation failed. Check the usm_type and queue."
+            msg = "In {0:s}.wrap(). USM allocation failed at {1:s}(). Check the usm_type and queue.".format(
```

Comment:
Please use f-string and look out for 80 char libe width .

Reply:
Fixed.